nomad/stream/event_broker.go

```diff
@@ -367,6 +367,14 @@ func aclAllowsSubscription(aclObj *acl.ACL, subReq *SubscribeRequest) bool {
 			if ok := aclObj.AllowNsOp(subReq.Namespace, acl.NamespaceCapabilityHostVolumeRead); !ok {
 				return false
 			}
+		case structs.TopicCSIVolume:
+			if ok := aclObj.AllowNsOp(subReq.Namespace, acl.NamespaceCapabilityCSIReadVolume); !ok {
+				return false
+			}
+		case structs.TopicCSIPlugin:
+			if ok := aclObj.AllowNsOp(subReq.Namespace, acl.NamespaceCapabilityReadJob); !ok {
```

Comment:
was `ReadJob` selected because the `CSIPlugin` contains alloc information (and whatever `JobDescriptions` is)?

Reply:
Yes. Although the reason we're using it here is because the `CSIPlugin.Read` RPC handler uses the same permission, and they should match :grin: